Sources/Curassow/Worker.swift

```diff
@@ -86,7 +86,7 @@ class WorkerTemp {
 #if os(Linux)
     let filename = Data(capacity: Int(PATH_MAX))
     let pointer = filename.bytes.bindMemory(to: CChar.self, capacity: filename.capacity)
-    let size = readlink("/proc/self/fd/\(descriptor)", pointer, filename.capacity)
+    let _ = readlink("/proc/self/fd/\(descriptor)", pointer, filename.capacity)
```

Comment:
Would probably make sense to check if the operation succeeded instead of silently handling the error.

What about changing this to something like:

```swift
let size = readlink("/proc/self/fd/\(descriptor)", pointer, filename.capacity)
if size == -1 {
  fatalError("readLink failed, errno: \(errno)")
}
```